src/components/ChatBox.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import ChatBox, { ChatBoxProps } from './ChatBox';
+
+export default {
+  title: 'Components/ChatBox',
+  component: ChatBox,
+  argTypes: {
+    message: { control: 'text' },
+    width: { control: 'number' },
+    height: { control: 'number' },
+    animationDuration: { control: 'number' },
+    variant: { control: 'radio', options: ['default', 'flip'] }, // Added variant control
```

Comment:
Variant Documentation
You added the 'variant' control with 'default' and 'flip' as options. It might be useful to include a short comment or additional description in the story to explain visually or functionally what 'flip' does differently from 'default'.